config/database.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const db = {
+  host: process.env.MONGO_HOST || 'localhost',
+  port: process.env.MONGO_PORT || 27017
+};
+
+const options = {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  serverSelectionTimeoutMS: 5000 // Keep trying to send operations for 5 seconds
+};
+
+const init = async () => {
+  /* MONGOOSE SETUP */
+  const connectionString = `mongodb://${db.host}:${db.port}/blog_everyone:`;
+
+  await mongoose.connect(connectionString, options).catch(() => {
```

Comment:
don't mix async/await with .catch/.then handlers.
If you use async/await you handle errors with  try/catch blocks :)

Reply:
fixed